deepspeed/runtime/engine.py

```diff
@@ -622,8 +622,12 @@ def _configure_basic_optimizer(self, model_parameters):
             from deepspeed.ops.lamb import FusedLamb
             optimizer = FusedLamb(model_parameters, **optimizer_parameters)
         elif self.optimizer_name() == ONEBIT_ADAM_OPTIMIZER:
-            from deepspeed.runtime.fp16.onebit_adam import OnebitAdam
+            from deepspeed.runtime.fp16.onebit.onebitadam import OnebitAdam
```

Comment:
the filename can remain adam instead of onebitadam, right? The class name OnebitAdam is good but let's keep the filename as adam.py instead of onebitadam.py. It looks redundant when we are inside the onebit folder.

Reply:
just clarifying: I want it to look like from ```deepspeed.runtime.fp16.onebit.adam import OnebitAdam``` instead of ```from deepspeed.runtime.fp16.onebit.onebitadam import OnebitAdam```

Reply:
Agree. Will change it.